client/src/components/Landing/Card.scss

```diff
@@ -12,7 +12,9 @@
     }
 
     @media screen and (min-width: 576px) {
-      width: 100%;
+      width: 90%;
+      padding: 15px;
```

Comment:
padding should not change on smaller screens - width should also be fined at 100%, the side padding should be done on a container higher in the hierarchy